src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
   // write your code here
-}
+  const result = {};
+  const sourceArray = sourceString.split(';');
```

Comment:
It is better to handle possible trailing semicolon in the sourceString before splitting it. Otherwise, you may end up with an empty string as the last element of the array, which will create an unnecessary property in the resulting object. You can use trim() function to remove trailing semicolon.

Reply:
fixed